e2e/client/playwright/page-object-models/monitoring.ts

```diff
@@ -29,9 +29,17 @@ export class Monitoring {
         const actionsWithoutLast = actionPath.slice(0, actionPath.length - 1);
 
         for (const action of actionsWithoutLast) {
-            await this.page.locator(s('context-menu')).getByRole('button', {name: action}).hover();
+            await this.page.locator(s('context-menu')).getByRole('button', {name: action, exact: true}).hover();
         }
 
-        await this.page.locator(s('context-menu')).getByRole('button', {name: actionPath[actionPath.length - 1]}).click();
+        await this.page.locator(s('context-menu'))
+            .getByRole('button', {name: actionPath[actionPath.length - 1], exact: true})
+            .click();
+    }
+
+    async createArticleFromTemplate(template: string): Promise<void> {
+        await this.page.locator(s('content-create')).click();
+        await this.page.locator(s('default-desk-template')).click();
```

Comment:
Create article from template means that you would be able to pass a template name and not use a default desk template. Also don't modify slugline in this method.